src/defs/policyResultRepresentation.ts

```diff
@@ -0,0 +1,9 @@
+import PolicyRepresentation, {DecisionEffect} from './policyRepresentation';
+import ScopeRepresentation from './scopeRepresentation';
+
+export default interface PolicyResultRepresentation {
+  policy?: PolicyRepresentation;
+  status?: DecisionEffect;
+  associatedPolicies?: PolicyResultRepresentation[];
+  scopes?: ScopeRepresentation[];
```

Comment:
According to the code you linked this should be `string[]`, or is there some transformation going on to link the full `ScopeRepresentation`? See the source code [here](https://github.com/keycloak/keycloak/blob/bfce612641a70e106b20b136431f0e4046b5c37f/core/src/main/java/org/keycloak/representations/idm/authorization/PolicyEvaluationResponse.java#L167).

Reply:
Ah, nice catch! Yes - fixed.